src/lib/UI/Config/Mapper/FieldType/RichText/CustomTag.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\UI\Config\Mapper\FieldType\RichText;
+
+use EzSystems\EzPlatformAdminUi\UI\Config\Mapper\FieldType\RichText\CustomTag\AttributeMapper;
+use EzSystems\EzPlatformAdminUi\UI\Translation\RichTextCustomTagsTranslationExtractor;
+use Symfony\Component\Asset\Packages;
+use Symfony\Component\Translation\TranslatorInterface;
+use Traversable;
+
+/**
+ * RichText Custom Tag configuration mapper.
+ */
+class CustomTag
+{
+    /** @var array */
+    private $customTagsConfiguration;
+
+    /** @var TranslatorInterface */
+    private $translator;
+
+    /** @var \Symfony\Component\Asset\Packages */
```

Comment:
Please use imported names as this is the approach used in AdminUI.

Reply:
Done, also for other occurrences.